### ISSUE TITLE
"Code Size" means different things in the paper vs in the documentation?

### ISSUE THREAD

**Issue by uduse** [facebookresearch/CompilerGym#718]
Doc says "code size" is text size
![image](https://user-images.githubusercontent.com/4717005/174687035-e43e07d5-0e15-4379-808b-9b50883fc81d.png)

Paper says "code size" is IR size, "binary size" is text size
![image](https://user-images.githubusercontent.com/4717005/174687068-0ca985a4-7018-48e3-8c60-288fada3c104.png)

**Comment by ChrisCummins** [comment]
Good catch. I'd suggest renaming the header in the documentation site to `Object File Size`, which is less ambiguous than "codesize".

Cheers,
Chris
